packages/gatsby-theme-docs/src/layouts/internals/reset.js

```diff
@@ -0,0 +1,358 @@
+        details,
+        embed,
+        figure,
+        figcaption,
+        footer,
+        header,
+        hgroup,
+        menu,
+        nav,
+        output,
+        ruby,
+        section,
+        summary,
+        time,
+        mark,
+        audio,
+        video {
+          margin: 0;
+          padding: 0;
+          border: 0;
+          font-size: 100%;
+          vertical-align: baseline;
+          min-width: 0; /* flexbox defaults to auto causing confusion*/
+        }
+        /* HTML5 display-role reset for older browsers */
+        article,
+        aside,
+        details,
+        figcaption,
+        figure,
+        footer,
+        header,
+        hgroup,
+        menu,
+        nav,
+        section {
+          display: block;
+        }
+        ol,
+        ul {
+          list-style: none;
+        }
+        blockquote,
+        q {
+          quotes: none;
+        }
+        blockquote:before,
+        blockquote:after,
+        q:before,
+        q:after {
+          content: '';
+          content: none;
+        }
+        table {
+          border-collapse: collapse;
+          border-spacing: 0;
+        }
+
+        /* Change from 'box-sizing: content-box' so that 'width' is not affected by 'padding' or 'border'. */
+        *,
+        *::before,
+        *::after {
+          box-sizing: border-box;
+        }
+
+        body {
+          margin: 0;
+          padding: 0;
+          border: 0;
+          font-style: normal;
+          font-variant: normal;
+          font-stretch: normal;
+          font-family: 'Roboto', -apple-system, system-ui, BlinkMacSystemFont,
+            'Segoe UI', 'Helvetica Neue', Arial, sans-serif;
+          font-weight: ${typography.fontWeights.regular};
+          font-size: ${typography.fontSizes.body};
+          line-height: ${typography.lineHeights.body};
+          text-align: left;
+          vertical-align: baseline;
+          color: ${colors.light.primary};
+
+          /* Allow Browsers to invest more CPU into text rendering.
+          We do NOT apply '-moz-osx-font-smoothing: grayscale' and '-webkit-font-smoothing: antialiased'.
+          They worsen the experience on non-retina screens.*/
+          text-rendering: optimizelegibility;
+
+          /* Prevent adjustments of font size after orientation changes in IE on Windows Phone and in iOS.*/
+          -webkit-text-size-adjust: 100%;
+          /* Change the default tap highlight to be completely transparent in iOS. */
+          -webkit-tap-highlight-color: rgba(black, 0);
+        }
+
+        /* See https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible
+          and https://developer.paciellogroup.com/blog/2018/03/focus-visible-and-backwards-compatibility/
+          and the boostrap reboot source code */
+        [tabindex='-1']:focus:not(:focus-visible) {
+          outline: 0 !important;
+        }
+
+        /* Default typography for typical  monospace elements */
```

Comment:
We can move this to the `Global` component, since we are explicitly defining some values here.